examples/dev/metadata.eg.ts

```diff
@@ -8,6 +8,7 @@
  * even plain-text descriptions. This metadata serves as the basis off of which we
  * generate chain-specific APIs. Unless you are building an advanced Capi-based library,
  * chances are that you don't need to work with the metadata directly.
+ * @test_skip node
```

Comment:
```suggestion
 * @test_skip
```

Let's actually skip this in all test envs